### DISCUSSION TITLE
gremlin使用问题

### DISCUSSION THREAD

**Discussion by banbsyip** [alibaba/GraphScope#1334] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/18277076/155698366-94409694-fb4c-4bd7-9aec-843b39bb943c.png)
当我用gremlin进行负采样查询后，要怎么把负采样的结果导出来？
**Environment (please complete the following information):**
 - GraphScope version:  v0.10.0
 - OS: [ Linux]

**Comment by sighingnow** [comment, marked as answer]
Hi @banbsyip ,

You could refer to the examples [in our documentation](https://graphscope.io/docs/interactive_engine.html), and [the official gremlin-python docs](https://tinkerpop.apache.org/docs/current/reference/#gremlin-python-scripts).
